src/main/kotlin/at/bitfire/dav4jvm/okhttp/exception/HttpException.kt

```diff
@@ -23,7 +22,7 @@ open class HttpException(
     override val statusCode: Int,
     requestExcerpt: String?,
     responseExcerpt: String?,
-    errors: List<Error> = emptyList()
+    errors: List<at.bitfire.dav4jvm.Error> = emptyList()
```

Comment:
Why not import it here? Like its done in other places?

Reply:
Yes, that was not consistent, I corrected this in all places.